test/sslapitest.c

```diff
@@ -6118,6 +6172,7 @@ int setup_tests(void)
 #else
     ADD_ALL_TESTS(test_tls13_psk, 4);
 #endif  /* OPENSSL_NO_PSK */
+    ADD_ALL_TESTS(test_tls13_key_exchange, 2);
```

Comment:
Probably this should be inside `OPENSSL_NO_TLS1_3` guards. The test doesn't make sense without TLSv1.3

Reply:
Currently this is under `#ifndef OPENSSL_NO_TLS1_3`, I hope this is fine. Please clarify me if anything else is needed.

Reply:
Ah, sorry I misread the diff.